about_regular_expressions.exs

```diff
@@ -0,0 +1,36 @@
+defmodule AboutRegularExpressions do
+  use Koans
+
+  think "A simple regular expression" do
+    regex = Regex.compile!("This [W|w]orks!")
```

Comment:
I wonder if it'd be more idiomatic to use the `~r//` syntax for the initial example. Then maybe show that the delimiter can be changed. Then show that they can be compiled dynamically from strings.